tools/data/skeleton/gen_ntu_rgbd_bone.py

```diff
@@ -0,0 +1,64 @@
+import os
```

Comment:
Why do we need another script for bone feature extraction? This can not be conducted by `gen_ntu_rgbd_raw.py`?